modules/reference/pages/feature/requestTiming/examples.adoc

```diff
@@ -2,47 +2,46 @@
 == Examples
 
 === Configure Slow and hung request thresholds
-The following example sets the slow and hung request thresholds for all requests. The `requestTiming` element detects requests as slow if the requests require more than 10 seconds to complete, or as hung if the requests do not complete in 10 minutes:
+By default, the `requestTiming` element detects requests as slow if the requests require more than 10 seconds to complete, or as hung if the requests do not complete in 10 minutes. The following example sets the slow request threshold to 5 seconds and sets the hung request threshold to 5 minutes for all requests:
 [source,xml]
 ----
-<requestTiming slowRequestThreshold="10s" hungRequestThreshold="10m">
+<requestTiming slowRequestThreshold="5s" hungRequestThreshold="5m">
 </requestTiming>
 ----
 
 === Configure JDBC request thresholds
-The following example sets the slow and hung JDBC request thresholds within the `requestTiming` element configuration. The `jdbcTiming` element detects that the JDBC request is slow if the request requires more than 500 ms to complete, or hung if the request does not complete in 5 minutes:
+The following example sets the slow and hung JDBC request thresholds within the `requestTiming` element for any JDBC data source that is specified with the `dataSourceName` attribute. The `jdbcTiming` element detects that the JDBC request is slow if the request requires more than 500 ms to complete, or hung if the request does not complete in 5 minutes:
 [source,xml]
 ----
-<requestTiming slowRequestThreshold="10s" hungRequestThreshold="10m">
+<requestTiming slowRequestThreshold="5s" hungRequestThreshold="5m">
   <jdbcTiming dataSourceName="testDBSource" slowRequestThreshold="500ms" hungRequestThreshold="5m"/>
 </requestTiming>
 ----
-The `jdbcTiming` element configuration always overrides the `requestTiming` element configuration for the slow and hung request thresholds. The `jdbcTiming` element configuration can be only used to time a root level request.
+The `jdbcTiming` element configuration always overrides the `requestTiming` element configuration for the slow and hung request thresholds. The `jdbcTiming` element configuration can be only used to time a root-level request. For example, if a servlet request makes a JDBC query, the JDBC query cannot be timed because the servlet request is the root level request.
```

Comment:
the servlet request is the root level request
--->
the servlet request is the root-level request